doc/releases/changelog-dev.md

```diff
@@ -368,6 +342,8 @@ Users can specify the control wires as well as the values to control the operati
   Stop using `op._wires = new_wires`, use `qml.map_wires(op, wire_map=dict(zip(op.wires, new_wires)))`
   instead.
   [(#3186)](https://github.com/PennyLaneAI/pennylane/pull/3186)
+* Added a new `pennylane.tape.QuantumScript` class that contains all the non-queuing behavior of `QuantumTape`. Now `QuantumTape` inherits from `QuantumScript` as well as `AnnotatedQueue`.
```

Comment:
Maybe this goes in breaking changes? 🤔

Reply:
Maybe? Since `QuantumTape` inherits from `QuantumScript` you should still be able to access all attributes and methods as normal. But if someone was looking to write their own logic (more developer facing) then its in a different place.

Reply:
Hmm okay good point